docs/application/native/guides/ui/efl/efl-util.md

```diff
@@ -247,7 +247,7 @@ To take a screenshot:
 
 To generate key input events:
 
-1. Create the `efl_util_inputgen_h` structure and initialize the structure members with the `efl_util_input_initialize_generator()` or `efl_util_input_initialize_generator_with_name()` function:
+1. Create the `efl_util_inputgen_h` structure and initialize the structure members with the `efl_util_input_initialize_generator()` , `efl_util_input_initialize_generator_with_name()` , or `efl_util_input_initialize_generator_with_sync()` function:
```

Comment:
Please remove one space before each comma. 

`efl_util_input_initialize_generator()`, `efl_util_input_initialize_generator_with_name()`, or `efl_util_input_initialize_generator_with_sync()` function

Reply:
I have asked @jhyuni to add one comma before "or".

Reply:
Hello, @Reni-90 
And I'm asking @jhyuni to remove the **space** in front of comma, not comma itself.
We have additional spaces in front of each comma.

Reply:
I am sorry. 
I suggested the "comma issue". I think @jhyuni  misinterpreted. Sorry for that as well.